client/app/components/saveable-form.hbs

```diff
@@ -0,0 +1,22 @@
+{{!-- template-lint-disable no-log --}}
```

Comment:
Is this supposed to be logging or should it be removed?

Reply:
I kinda want it to, for helping with any debugging going on.